develop/learning-paths/migration/articles/01-intro/00-developer-journey-from-liferay-6-to-liferay-7-intro.markdown

```diff
@@ -0,0 +1,15 @@
+# Developer Journey from Liferay 6 to Liferay 7
+
+You've been using Liferay as a developer for some time now; you've read
+documentation and blogs and have maybe even gone through some training. More
+importantly, you have already developed some portlets and extensions. And now
+you hear about all the changes in Liferay 7 and you wonder: will I have to
```

Comment:
I think we need something more "punchy" to start this out. This sounds a little bit like the beginning to a Choose Your Own Adventure book. :-)